htslib/hts.h

```diff
@@ -457,14 +457,15 @@ char *hts_format_description(const htsFormat *format);
   @param fn       The file name or "-" for stdin/stdout. For indexed files
                   with a non-standard naming, the file name can include the
                   name of the index file delimited with HTS_IDX_DELIM
-  @param mode     Mode matching / [rwa][bceguxz0-9]* /
+  @param mode     Mode matching / [rwa][bcefguxz0-9]* /
```

Comment:
What's the x and e in this string?  I can't see any support for these in the code, and they're undocumented below.  Some legacy thing?

Reply:
They are passed through to `hopen()` and interpreted there (see #245), and actually are documented in the `hts_open()` blurb — look further below.

Reply:
I should have opened the file to look.  :)

I saw them documented in alphabetical order and github trimmed the surrounding text at just the wrong spot.  Duh!  Sorry and thanks for the explanation.

I'll be adding bits of this as I now have the go ahead for doing the writing support too, which will start from this.  Cheers.